syntax = "proto3";
package market.proto;

message Quantity {
  string kind = 1;
  int64 amount = 2;
}

message Container {
  map<string, int64> quantities = 1;
}

message TradeGood {
  string name = 1;  // required
  int64 decay_rate_u = 2;
  int64 bulk_u = 3;
  int64 weight_u = 4;
}

